dts/arm/st/h7/stm32h7.dtsi

```diff
@@ -712,6 +712,19 @@
 			label = "DMA_2";
 		};
 
+		dmamux1: dmamux@40020800 {
+			compatible = "st,stm32-dmamux";
+			#dma-cells = <4>;
+			reg = <0x40020800 0x400>;
+			interrupts = <102 0>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB1 0x00000001>;
```

Comment:
Could we have a comment to explain that DMAMUX has no specified clock on the AHB